core/src/test/scala/com/evolutiongaming/catshelper/TestIORuntime.scala

```diff
@@ -0,0 +1,7 @@
+package com.evolutiongaming.catshelper
+
+import cats.effect.unsafe.IORuntime
+
+trait TestIORuntime {
```

Comment:
why do we need this trait at all? writing single line of 

```
implicit val ioRuntime: IORuntime = cats.effect.unsafe.IORuntime.global
```
is more or less equivalent to 

```
extends TestIORuntime
```

I'd personally prefer firs option.

Reply:
I agree, this is a result of multiple steps of tinkering with test code, will remove that

Reply:
done